test/iframe.test.ts

```diff
@@ -0,0 +1,26 @@
+import { getByTestIdSelector, Ivya } from '../src'
+import { expect, test } from 'vitest'
+
+test('locates element inside iframe', async () => {
+  const button = document.createElement('button')
+  button.textContent = 'Click me'
+
+  const iframe = document.createElement('iframe')
+  iframe.setAttribute('data-testid', 'test-iframe')
+  iframe.srcdoc = button.outerHTML
+  document.body.appendChild(iframe)
+
+  const ivya = Ivya.create({
+    browser: 'chromium',
+  })
+
+  // wait for iframe to load
+  await new Promise((resolve) => (iframe.onload = resolve))
+
+  const iframeSelector = getByTestIdSelector('data-testid', 'test-iframe')
+  expect(ivya.queryLocatorSelector(iframeSelector)).toBe(iframe)
+
+  expect(
+    ivya.queryLocatorSelector('iframe >> internal:control=enter-frame >> button')
+  ).toEqual(button)
```

Comment:
This seems very wrong, the button was never added to anything. Using `toEqual` here is highly suspicious because iframe obviously has a different button element (the reference is not the same), just select the element with `document.querySelector` on the iframe instead of referencing the `button` element here